### DISCUSSION TITLE
Adding CAST to R-Instat

### DISCUSSION THREAD

**Discussion by rdstern** [IDEMSInternational/R-Instat#9801] [Ideas]
This will later split into a number of issues.  I have put the 0.5.0 milestone so we consider how to start with CAST - not that the addition will finish by then.

1. How will we install CAST?  Will we have information on the AMI and/or R-Instat installation site?
2. Can we add a call to CAST to the Instat help menu.  I understand Doug has added the facility to go to even a page, so what could we do from the help menu?  We could have a dialogue to go perhaps directly to any book of our choice?
3. I believe there is (or could easily be) a package of CAST data sets, from any book.  Could we produce a package (or multiple packages) and then have another option on our File > Open from Library to access these sets, just as we do for data from the standard packages?
 4. Can we extend our help on the appropriate dialogues to go to one (or more) relevant pages in CAST.  Obvious examples are boxplots, bar charts, pie charts and some of the tables dialogues.
5. What else?

**Comment by lilyclements** [comment]
@rdstern is this something we still want to add into R-Instat?
